SebastienBerube/dxc-hlsl-2-spirv-test
# Feature/add dxc libs

**Quick Description**
Add dxcompiler libraries

**How to test:**
- Run cmake
- Open solution project with visual studio
- Build and run project 'dxc_test_main'
- See compilation errors in the console (this is the intended behaviour)
